applications/trilinos_application/custom_processes/trilinos_variational_distance_calculation_process.h

```diff
@@ -245,66 +211,59 @@ class TrilinosVariationalDistanceCalculationProcess
         p_distance_model_part->Conditions().clear();
         p_distance_model_part->Elements().clear();
 
-        p_distance_model_part->SetProcessInfo(  base_model_part.pGetProcessInfo() );
+        p_distance_model_part->SetProcessInfo(base_model_part.pGetProcessInfo());
         p_distance_model_part->SetBufferSize(base_model_part.GetBufferSize());
         p_distance_model_part->SetProperties(base_model_part.pProperties());
         p_distance_model_part->Tables() = base_model_part.Tables();
 
-        //assigning the nodes to the new model part
+        // Assigning the nodes to the new model part
         p_distance_model_part->Nodes() = base_model_part.Nodes();
 
-        //ensure that the nodes have distance as a DOF
-        for (ModelPart::NodesContainerType::iterator iii = base_model_part.NodesBegin(); iii != base_model_part.NodesEnd(); iii++)
-        {
-            iii->AddDof(DISTANCE);
-        }
+        // Ensure that the nodes have distance as a DOF
+        VariableUtils().AddDof<Variable<double> >(DISTANCE, base_model_part);
```

Comment:
shouldn't dofs now always have reactions?
@loumalouomega

Reply:
If I'm not wrong, there is no problem.